src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
So it looks like the `CA1510` errors in CI come from moving this to a TargetFramework that understands what FreeBSD is. There are NoWarns for `CA1510` for TargetFramework that is `netstandard` or `NetFramework` but not something like `net6.0`

Reply:
`netstandard2.0` (meta TFM) is correct. [It implies](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0) all .NET Core versions > 2.0 and .NET versions. Without this, HostModel will not be loaded for apps built against versions 2.0-6.0. What happens if we don't make this change?

Reply:
So leaving it as `netstandard2.0` and reverting [src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs](https://github.com/dotnet/runtime/pull/80323/files#diff-b6fd7c2b3db683d621e52e7ecec98bb9b5a6dab57042df8daa1b675b897e14b9) (remove FreeBSD references) produces no error when building locally (`./build.sh -c Release -ci --os FreeBSD /p:OfficialBuildID=20230109.99 /p:PublishReadyToRun=false`) 

I have been using some variation of these changes, just for local/private builds, since around net5.0 and I don't recall why I even made the change in the first place.

both files will need to be reverted:
just reverting to `netstandard2.0` but leaving the changes in TargetInfo.cs causes the build to fail with 
```
/root/runtime/src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs(100,147): error CS0117: 'OSPlatform' does not contain a definition for 'FreeBSD' [/root/runtime/src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj]
```
Sadly, FreeBSD showed up only at [netcore3.0 or higher](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform?view=netcore-3.0). 
[netstandard2.0](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform?view=netstandard-2.0) knows nothing of it.

Reply:
You can use `new OSPlatform("FREEBSD")` instead.